.gitignore

```diff
@@ -26,3 +26,5 @@
 .envrc
 /inventory.yaml
 SSH_I_DUMPED
+/spec/fixtures/acceptance-credentials.conf
+/spec/fixtures/acceptance-device.conf
```

Comment:
These will need added to the .sync.yml (https://github.com/puppetlabs/cisco_ios/blob/master/.sync.yml#L4) otherwise when we next perform a `pdk update` they'll get removed.